npm/webpack-dev-server-fresh/src/devServer.ts

```diff
@@ -137,6 +138,10 @@ devServer.create = async function (devServerConfig: WebpackDevServerConfig) {
       case 'vue-cli':
         frameworkConfig = vueCliHandler({ devServerConfig, sourceWebpackModulesResult })
         break
+
```

Comment:
Just a style thing, with big switch statements like this, I always prefer having them out in their own function, so you can just `return` from each branch, a'la
```
getConfig = async (framework) => {
  switch (framework) {
    case 'react':
      return
    case 'nuxt':
      return await nuxtHandler({ devServerConfig, sourceWebpackModulesResult })
    ...etc...
  }
}

let frameworkConfig = await getConfig(devServerConfig.framework)
```

Reply:
Addressed in ac29f02aa37dc2f16e412f546093b9f48e0cf710